sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_helpers.py

```diff
@@ -2,40 +2,41 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-
 import json
+import logging
+from typing import Generator, List
 import zlib
 
-MAX_CHUNK_SIZE = 1000000
-CHAR_SIZE = 4
+_LOGGER = logging.getLogger(__name__)
+
+MAX_CHUNK_SIZE_BYTES = 1024 * 1024 # 1 MiB
+CHAR_SIZE_BYTES = 4
 
 
-def _split_chunks(logs):
-    chunks = []
+def _split_chunks(logs: List) -> Generator:
```

Comment:
Do we have a more comprehensive type hint for these `logs`? Presumably the public facing API uses something more concrete?

Reply:
Looks like we're using the defined `JSON` type - in which case these should also be:
`_split_chunks(logs: List[JSON]) -> Generator[JSON]`